### ISSUE TITLE
Enforce usage of 64bit client

### ISSUE THREAD

**Issue by kasoki** [kasoki/LoginWars2#12]
If ArenaNet will ship both "Gw2.exe" and "Gw2-64.exe" in the future it might be interesting to always use the 64bit executable if the user can use it.

This shouldn't be implemented before the 64bit client leaves beta though.

**Comment by KairuByte** [comment]
I would disagree on the enforcing. There are many valid reasons to use 32 bit on a 64 bit system. Performance is one of them.

**Comment by atomicptr** [comment]
The idea is more like pick the 64bit one by default if the system allows it, obviously this wouldn't take away your ability to re-select the 32bit version.
